tools/build_defs/framework.bzl

```diff
@@ -136,6 +136,7 @@ Instances of ForeignCcArtifact are incapsulated in a depset ForeignCcDeps#artifa
         "bin_dir_name": "Bin directory, relative to install directory",
         "lib_dir_name": "Lib directory, relative to install directory",
         "include_dir_name": "Include directory, relative to install directory",
+        "declared_outputs": "List of files that must be generated",
```

Comment:
Why do we need it here?

ext_buld_dirs are gathered to copy directories, built with this rule, under the $EXT_BULD_DEPS
It is enough then; all the contents with all libraries and include directories etc is already there; why do we need to have files as well?